apps/builder/app/builder/features/sidebar-left/panels/pages/custom-metadata.tsx

```diff
@@ -0,0 +1,173 @@
+import {
+  Button,
+  Grid,
+  InputErrorsTooltip,
+  InputField,
+  Label,
+  SmallIconButton,
+  Text,
+  theme,
+} from "@webstudio-is/design-system";
+import { DeleteIcon, PlusIcon } from "@webstudio-is/icons";
+import { useId } from "react";
+
+type Meta = {
+  property: string;
+  content: string;
+};
+
+type CustomMetadataProps = {
+  customMetas: Meta[];
+  onChange: (value: Meta[]) => void;
+};
+
+const PropertyContent = (props: {
+  property: string;
+  content: string;
+  onDelete: () => void;
+  onChange: (property: string, content: string) => void;
+}) => {
+  const propertyId = useId();
+  const contentId = useId();
+
+  return (
+    <Grid
+      gap={2}
+      css={{
+        gridTemplateColumns: `${theme.spacing[18]} 1fr 19px`,
+        gridTemplateAreas: `
+      "property property-input button"
+      "content content-input button"
+    `,
+      }}
+      align={"center"}
+    >
+      <Label htmlFor={propertyId} css={{ gridArea: "property" }}>
+        Property
+      </Label>
+      <InputErrorsTooltip errors={undefined}>
+        <InputField
+          css={{ gridArea: "property-input" }}
+          tabIndex={1}
+          id={propertyId}
+          property="path"
+          value={props.property}
+          onChange={(event) => {
+            props.onChange(event.target.value, props.content);
+          }}
+        />
+      </InputErrorsTooltip>
+      <Label htmlFor={contentId} css={{ gridArea: "content" }}>
+        Content
+      </Label>
+      <InputErrorsTooltip errors={undefined}>
+        <InputField
+          css={{
+            gridArea: "content-input",
+          }}
+          tabIndex={1}
+          id={contentId}
+          property="path"
+          value={props.content}
+          onChange={(event) => {
+            props.onChange(props.property, event.target.value);
+          }}
+        />
+      </InputErrorsTooltip>
+      <Grid
+        css={{
+          gridArea: "button",
+          justifyItems: "center",
+          gap: "2px",
+          color: theme.colors.foregroundIconSecondary,
+        }}
+      >
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          width="19"
+          height="11"
+          viewBox="0 0 19 11"
+          fill="currentColor"
+        >
+          <path d="M10 10.05V6.05005C10 2.73634 7.31371 0.0500488 4 0.0500488H0V1.05005H4C6.76142 1.05005 9 3.28863 9 6.05005V10.05H10Z" />
+        </svg>
```

Comment:
Why inline?

Reply:
It's not named icon in design, so inlined